peps/pep-0730.rst

```diff
@@ -241,6 +235,44 @@ unique Framework name by using the full module name as the framework name (i.e.,
 ``foo.bar._whiz.framework``). A framework is a directory; the finder will look
 for ``_whiz.dylib`` in that directory.
 
+Compilation
+-----------
+
+The only binary format that will be supported is a dynamically-linkable
+``libPython.dylib``, packaged in an iOS-compatible framework format. While the
+``--undefined dynamic_lookup`` compiler option currently works, the long-term
+viability of the option cannot be guaranteed. Rather than rely on a compiler
+flag with an uncertain future, binary modules on iOS will be linked with
+``libPython``. This, in turn, makes statically-linkable builds of
+``libPython.a`` impractical.
+
+Building CPython for iOS requires the use of the cross-platform tooling in
+CPython's ``configure`` build system. A single ``configure``/``make``/``make
+install`` pass will produce a ``Python.framework`` artefact that can be used on
+a single ABI and architecture.
+
+Additional tooling will be required to merge the ``Python.framework`` builds for
+multiple architectures into a single "fat" library. Tooling will also be
+required to merge multiple ABIs into the ``XCframework`` format that Apple uses
+to distribute multiple frameworks for different ABIs in a single bundle.
+
+An Xcode project will be provided for the purpose of running the CPython test
+suite. Tooling will be provided to automate the process of compiling the test
+suite binary, start the simulator, install the test suite, and execute it.
+
+Distribution
+------------
+
+Adding iOS as a Tier 3 platform only requires adding support for compiling an
+iOS-compatible build from an unpatched CPython code checkout. It does not
+require production of officially distributed iOS artefacts for use by end-users.
+
+If/when iOS is updated to Tier 2 or 1 support, the tooling used to generate an
+``XCframework`` package could be used to produce an iOS distribution artefact.
+This could then be distributed as an "embedded distribution" analogous to the
+Windows embedded distribution, or as a Cocoapod or Swift Package that could be
```

Comment:
```suggestion
Windows embedded distribution, or as a CocoaPods or Swift package that could be
```

Reply:
Agreed that the capitalization on "CocoaPod" is incorrect... but I'm not sure about the pluralisation. The project is "CocoaPods", but their [documentation](https://guides.cocoapods.org) refers to "Making a CocoaPod", so the singular would seem more appropriate.

Reply:
Good spot, I've changed the suggestion to singular.

Reply:
The updated version already incorporates these 2 changes